src/sendMessage.js

```diff
@@ -184,6 +186,26 @@ module.exports = function(defaultFuncs, api, ctx) {
     }
     cb();
   }
+  
+  function handleEmoji(msg, form, callback, cb) {
+    if (msg.emojiSize != null && msg.emoji == null) {
+      return callback({error: "emoji property is empty"});
+    }
+    if (msg.emoji) {
+      if (msg.emojiSize == null) {
+        msg.emojiSize = "medium";
+      }
+      if (msg.emojiSize != "small" && msg.emojiSize != "medium" && msg.emojiSize != "large") {
+        return callback({error: "emojiSize property is invalid"});
+      }
+      if ((form['body'] != null || form['body'] != "") == false) {
```

Comment:
This if statement seems a bit weird. You want to error when the body _is_ empty right?
So maybe do:
if (form['body'] != null && form['body'] != "") 
Because emojis are only valid when the body is empty